### ISSUE TITLE
Report generation fails when attempting to create png badge

### ISSUE THREAD

**Issue by tylerohlsen** [danielpalme/ReportGenerator#224]
When attempting to create badges, the png creation fails when attempting to find _System.Memory_.

NuGet package reference from csproj
`<DotNetCliToolReference Include="dotnet-reportgenerator-cli" Version="4.0.15" />`  

Command line:  
```
dotnet reportgenerator -reports:%outputDir%\*.CodeCoverage.xml -targetdir:%outputDir%\CoverageReport -reporttypes:Badges;Html
```

```
 Creating summary
  Writing report file 'Build\TestOutput\CoverageReport\badge_linecoverage.svg'
  Writing report file 'Build\TestOutput\CoverageReport\badge_linecoverage.png'
  Error during rendering summary report (Report type: 'Badges'): Could not load file or assembly 'System.Memory, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified.
```

**Comment by danielpalme** [comment]
Perhaps related to https://github.com/SixLabors/ImageSharp/issues/459

**Comment by danielpalme** [comment]
Could you please try again with this new release:
https://www.nuget.org/packages/dotnet-reportgenerator-cli/4.1.0

```
<DotNetCliToolReference Include="dotnet-reportgenerator-cli" Version="4.1.0" />
```

**Comment by tylerohlsen** [comment]
Confirmed that it is fixed in 4.1.0!  Thank you!
